components/automate-chef-io/content/docs/on-prem-builder.md

```diff
@@ -46,6 +50,35 @@ Roughly 80 GB of the disk space is designated for Chef Automate; the rest is use
 Chef Habitat Builder and the artifacts it stores. The current implementation uses Minio for
 Chef Habitat artifact storage; we do not support using Artifactory for artifact storage.
 
+### Hardware Sizing Considerations
+
+#### Environment Variables
+The frequency at which Habitat Supervisors poll the on-premises Chef Builder API for changes can be controlled by three Environment variables in the Chef Habitat Supervisor's runtime.
+
+| Variable | Default | Description |
+| --- | --- | --- |
+| `HAB_SUP_UPDATE_MS` | 60000 | Frequency in milliseconds governing how often to check for Supervisor updates when running with the [--auto-update](https://www.habitat.sh/docs/habitat-cli/#hab-sup-run) flag. |
```

Comment:
Maybe "Number of milliseconds between checks for Supervisor updates when running ..."?